static/core-ui/utils.less

```
@@ -0,0 +1,10 @@
+
+// CSS containment variables.
+// The words after `but` explicitly show the excluded/incompatible values.
+@contain_all: layout size paint style;
+@contain_but_layout: size paint style;
+@contain_but_size: layout paint style;
+@contain_but_paint: layout size style;
+@contain_but_layout_size: paint style;
+@contain_but_layout_paint: size style;
+@containt_but_size_paint: layout style;
```

Nice work, mate. Small typo on this one (gotta fix the reference too).
```suggestion
@contain_but_size_paint: layout style;
```

Also, might want to use `except` instead of `but`, like `contain_except_layout_paint`, but not a big deal.

---

Thanks for the catch. I fixed this.

> Also, might want to use except instead of but, like contain_except_layout_paint, but not a big deal.

`but` seems easier to type 😃

---

`except` conveys exclusion better than `but`. Could we change this. @aminya

---

Renamed it in
https://github.com/atom/atom/pull/21561/commits/f37d7f8577724932e8e5b403cfc43ce355992da7